contracts/strategies/allocation/no-application/README.md

```diff
@@ -0,0 +1,38 @@
+Spec: No application with NFT contract
+---------------------------------
+
+## Overview 
+Some builders have flagged that they want the ability create pools that don't require applications, where anyone who has taken some action is automatically an eligible recipient in their pool. This is a strategy stub to test that format. 
+
+## Component Quickview
+As laid out in the [components overview](https://docs.google.com/document/d/1qoOP07oMKzUCyfb4HbnyeD6ZYEQa004i5Zwqoy7-Ox8/edit), each allocation strategy consists of some key components. This is a quick overview of the relevant components for this strategy.
+- **Recipient eligibility**: automatic
+- **Recipient approval**: programmatic
+- **Voter eligibility**: pool admin
+- **Voting method**: simple yes/no
+- **Calculation method**: simple yes/no
+- **Allocation shape**: discrete amount
+
+## Spec
+### Custom Variables
+This strategy will need the following custom variables:
+- `alternateRegistry` - location of alternate registry of eligible recipients
+
+### Standard Functions
+All standard functions are functions that the given user can call from the `Allo.sol` contract.
+#### `createPool()`
+The identity admin creates a new pool via `createPool`. At this time, the admin can set the following custom variables:
+- `alternateRegistry`
+    - Open question: should the identity admin also specify the logic by which the alternate projects should be accepted?
+
+When the pool is created, all recipients in the `alternateRegistry` (using the specified logic) should be added to the pool with the `Accepted` status.
```

Comment:
Why use a status at all in this scenario?

Reply:
That's a good question — there's a baked-in assumption that every recipient-pool pair on the protocol should have a status. We've typically thought about them as "application" statuses, but we're essentially trying to show what a given recipient's relationship to the pool is. 

- Null (no relationship)
- Pending (recipient has asked, admin has not answered yet)
- Rejected (recipient has asked, admin has said no)
- Accepted (recipient has asked, admin has said yes — recipient is in the pool)

Reply:
> they want the ability create pools that don't require applications

in this case nobody will ask and no admin will answer, correct?

for pools like this one the term `ApplicationStatus` is misleading, but as far as i see it:

`Null` unused
`Pending` unsused
only `Rejected` and `Accepted` are returned (does an address matches the criteria or not. eg: `balanceOf(address) >= treshold ? Accepted : Rejected`)

The bigger problem i see right now is:

````
    /// @notice Returns the status of the application
    /// @param applicationId The applicationId of the application
    function getApplicationStatus(uint256 applicationId) external view override returns (ApplicationStatus) {
        return applicationStatus[applicationId];
    }
````

`getApplicationStatus` (and other functions like `getPayout`) require an integer `applicationId` as parameter which is maybe not given in a case where no application is required. (Like discussed in our call yesterday)

Maybe we can use `bytes32` instead to cover all cases. For most strategies it doesn't matter if the `applicationId` is from type `uint256` or `bytes32`.
and in case of an `address` we could simply hash the address as identifier. 
what do you think @0xZakk ?